### ISSUE TITLE
Reflow Oven Build Guide

### ISSUE THREAD

**Issue by guevara** [guevara/read-it-later#4836]
Reflow Oven Build Guide<br>
<br>
https://ift.tt/2Z51Ov0<br>
<br>
<br>
<br>
<div>
<div width="786">
<p>This guide will show you how to build a great toaster reflow oven. It isn't difficult to build your own oven, and the result can be equal or better than commercially available ovens. Building it yourself will give you a better indication of what to look for, how it works and how to tune it to get excellent results.</p>
<p>The reflow oven build described here is fully capable of fine pitch lead-free soldering at very high temperatures, and can do it consistently and reliably. The entire tray can be filled with PCB's, and all will reflow perfectly. It uses Controleo3, a 6-relay programmable reflow oven controller.</p>
<p>This oven is designed to reflow lead-free solder paste with liquidus temperature around 217&deg;C (max temp around 250&deg;C). This is what is typically used by contract manufacturers making commercial PCB's. Typically, prototyping conditions should mimic the production environment to avoid mass-produced surprises. If you use solder paste with a lower liquidus temperature - some go as low as 138&deg;C - you can get away with a lesser oven.</p>
<p>If you're looking for the ControLeo2 build guide, click <a href="https://whizoo.com/c2">here</a>.</p>
<p><a href="https://whizoo.com/i/c3/b00.jpg"><img src="https://whizoo.com/i/c3/s00.jpg"></a></p>
To choose the right oven, a number of things need to be considered:
<ol>
<li>
<span>Size</span> - a small oven has less metal and air to heat up (less thermal mass). The oven in this guide is just 10 liters. Larger ovens require significantly more power to heat up. Get the smallest oven that will work for you.</li>
<li>
<span>Power</span> - a lot of power is required to get the right rate-of-rise at high temperatures. The oven in the guide is rated at 1150W. We add an extra heating element to bring the total power up to 1500W.</li>
<li>
<span>Type of element</span> - the elements should ideally be quick to heat up and cool down. <a href="https://cdnimg.webstaurantstore.com/images/products/extra_large/204679/937170.jpg">Resistive</a> heating elements are extremely slow to heat up and cool down, and make it difficult to follow a reflow profile. Infrared elements are usually a poor choice because they heat up dark components before lighter or shiny ones. Commercial ovens use infrared elements, but only to heat the air which is then blown onto the boards. The compromise is to use quartz infrared elements which emit less IR. They are typically shielded with a perforated steel sheet which limits direct radiation. Also, Controleo3 will limit how much the top element is used to further reduce uneven heating.</li>
<li>
<span>Ability to heat top and bottom independently</span> - some cheap ovens turn the top and bottom elements on together, with no way to control them individually. Avoid those - you need to be able to control where the heat is coming from.</li>
<li>
<span>Convection fan</span> - there is no doubt that a convection fan is a good thing. However, convection ovens are typically much larger and have less power for the volume. We have done a huge amount of testing on both oven types, and only recommend convection ovens if more PCB space is needed. Even then, you'll need a lot of insulation and for best performance your should consider buying a 2nd oven and donating its elements to the first. Consider using a low-temperature lead-free paste like Chip Quik SMDLTLFP10.</li>
<li>
<span>Available power</span> - Most USA outlets are rated at 110VAC 15A. This means that 110V x 15A = 1650W is available, though that should be derated by 20% if the oven is on continuously. The oven in this guide starts out at 1150W and a 350W boost element is added. This give a 10% safety margin, which is fine because it is rare that all elements are on at the same time during a reflow, and reflow durations are short. If you convert a convection oven, you may need a 20A outlet or a 240V circuit.</li>
</ol>
For this guide we found a suitable oven on Amazon.com. It was a brand new Black &amp; Decker T01303SB toaster oven for just $30 shipped! Black &amp; Decker sells many ovens with the same chassis (like the TO1322SBD, TO1332SBD and TO1313SBD), but with different controls, color or handle design so your oven may not look exactly like this one. It doesn't have a convection fan and the power consumption is 1150W. The "low" power consumption means we can add an additional heating element without blowing fuses. There are 2 heating elements - 600W on top and 550W on the bottom. The internal volume is small at 10 liters. The area for PCB's is 8"x10.5" (200mm x 260mm). Perfect!
<p>Controleo3 can control 6 relays. In this guide only 3 of the outputs are used: bottom element, top element and boost element. The door servo does not use any of the outputs; it is controlled separately.</p>
<p><span>Tip: click on any photo to see the high-resolution version.</span></p>
<p><a href="https://whizoo.com/i/c3/b01.jpg"><img src="https://whizoo.com/i/c3/s01.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b02.jpg"><img src="https://whizoo.com/i/c3/s02.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b03.jpg"><img src="https://whizoo.com/i/c3/s03.jpg"></a><br></p>
The oven in this guide may be unavailable in your area, or you may want to use a different oven - perhaps with a convection fan. A convection fan is nice to have but most convection ovens are larger, cost more and are more difficult to modify. The conductive properties of aluminum in the PCB tray and insulation (Floor &amp; Tunnel Shield II) is more effective at eliminating temperature differentials than moving air.
<p><span>Tips for Europeans</span> (and other 240V countries):</p>
<ul>
<li>Look at the forum discussion <a href="http://www.eevblog.com/forum/manufacture/diy-reflow-oven-suitable-ukeu-candidates/" target="_blank">here</a> and other customer's ovens <a href="https://whizoo.com/hof" target="_blank">here</a>.</li>
<li>The 6 considerations listed above are all relevant to your choice of oven.</li>
<li>Do not try to run a 110V oven through a 110V/240V converter! The power requirements are huge, as are the conversion losses. Use an oven designed to run on 240V.</li>
<li>Some 240V ovens use two 120V elements wired in series. This is fine if there are 2 bottom elements in series, or 2 top elements in series - but if you only have one element at the top and one at the bottom then they must be 240V elements. If the oven supports "top heating" and "bottom heating" then most likely they are 240V elements (good).</li>
<li>Do <span>not</span> connect 120V elements to a 240V supply! P = V<sup>2</sup>/R so wiring them this way will consume 4 times the power and likely burn out or melt the heating element.</li>
</ul>
<p><span>Tips for convection ovens</span>:</p>
<ul>
<li>Do not underestimate the effects of thermal mass! Larger oven are considerably more difficult to heat up. Consider buying two of the same oven, and using one to donate heating elements to the other. Make sure this will still work with your electric circuit.</li>
<li>Consider making a fake wall to reduce the internal volume. Eliminating the "pizza bulge" will help a lot.</li>
<li>A boost element should be added, but doing this will put the power draw beyond what a 15A circuit can provide. You may have access to a 20A circuit - great! If you don't add a boost element you'll need to go overboard with insulation. In addition to insulating the inside of the oven (Reflect-A-Gold and Floor &amp; Tunnel Shield II) you should also add some ceramic blanket insulation between the oven chamber and the outer shell.</li>
<li>The highest wattage elements are on the top - not good. It is best to remove all the elements, install the insulation and then replace them, with the highest wattage elements at the bottom. Best to cut the wires and splice them back together using a butt connector.</li>
<li>Use SSR's for the heating elements. The convection fan could be wired to be on all the time, or you can use a <a href="http://www.ebay.com/itm/261966674558" target="_blank">mechanical relay</a> or <a href="http://www.ebay.com/itm/271950871294" target="_blank">small SSR</a> to control it.</li>
<li>Add insulation (like ceramic fiber blanket) between the oven chamber and outer shell. This is a big oven, and you need all the help you can get to ensure uniform temperatures that can follow reflow profiles.</li>
<li>If your SSR's are switching large loads, consider adding passive (heat sink) or active (fan) cooling to them.</li>
<li>For the PCB tray, you can use a thin <a href="http://www.ebay.com/itm/321734966692" target="_blank">aluminum</a> or copper tray resting on the wire rack. Remove as much of the cross-members as you can to reduce thermal mass.</li>
<li>If you end up with an oven that is under-powered or under-insulated, consider using lead-free solder paste with a lower liquidus temperature (around 138&deg;C) like Chip Quik SMDLTLFP10.</li>
</ul>
If you are considering using a convection oven, then <a href="http://electronicdesign.com/blog/building-home-or-very-small-office-electronic-circuit-prototypes-part-2" target="_blank">blog 1</a>, <a href="https://sites.google.com/site/markscontroleo2build/" target="_blank">blog 2</a> and <a href="https://www.dropbox.com/sh/3i8kroljnjius7m/AAB3tk4eFyFAKjCFIO_O2B4Xa?dl=0" target="_blank">these photos</a> may help you. Common problems in home-made reflow ovens (or cheap commercial ovens) are hot and cold areas. A cold spot is an area that doesn't get enough heat to activate the flux at the right time, and allow the solder to reach liquidus temperature and reflow. A hot spot is an area that get too much heat and results in warped or blistered boards. Components can discolor, but some of the worst damage is what you can't see; the mean-time-before-failure (MTBF) of burnt IC's can be severely compromised.
<p><span>How does the boost element help?</span></p>
<ol>
<li>Heat comes from an additional location, helping to distribute the heat around the oven.</li>
<li>All elements can be run cooler, so there isn't an intensely concentrated heat source.</li>
<li>It is installed at the bottom of the oven, keeping heat away from sensitive components.</li>
</ol>
Good insulation is absolutely necessary when building your own reflow oven. It is almost impossible to have too much insulation. If you're using leaded solder paste and only reflowing a single small PCB at a time then it is possible without insulation. However, the goal in building this oven is to be able to use the entire oven tray for PCB's using high-temperature lead-free paste. Quality insulation capable of withstanding high temperatures is expensive but worth it. We recommend using <a href="http://www.designengineering.com/category/catalog/design-engineering-inc/heat-sound-barrier/reflect-gold" target="_blank">Reflect-A-Gold</a>, <a href="http://www.designengineering.com/category/catalog/boom-mat-acoustical-products/floor-tunnel-shield-ii-heat-sound-insulation" target="_blank">Floor &amp; Tunnel Shield II</a> and <a href="https://www.mcmaster.com/#93315k71/=18v8pwh" target="_blank">ceramic fiber blanket</a>.
<p>In this guide, Reflect-A-Gold and Floor &amp; Tunnel Shield II are used inside the oven to prevent heat loss to the steel oven walls. Ceramic fiber blanket insulation is added to the outside of the oven chamber (and inside the outer shell) to further reduce heat loss. We also use a high-temperature silicone glue to seal the air gaps between the oven panels and Nomex high-temperature gasket to reduce the air gap around the oven door.</p>
<p><span>Quality insulation is not cheap</span><br>
There is fake Reflect-A-Gold available on the Internet. It looks the like the real deal, and our testing shows the insulating properties to be very similar. However, the adhesive on this tape doesn't stand up to the high reflow temperatures as can be seen <a href="https://whizoo.com/i/c/l/rag1.jpg">here</a> and <a href="https://whizoo.com/i/c/l/rag2.jpg">here</a>. Some insulation (we're looking at you, <a href="https://www.heatshieldproducts.com/automotive/heat-shield-tape-and-fasteners/thermaflect-tape" target="_blank">Heatshield Products</a>) rated to higher temperatures than Reflect-A-Gold don't <a href="https://www.dvatp.com/tech/reflow_oven" target="_blank">work as well</a>.</p>
<p><span>How does insulation help?</span></p>
<ol>
<li>It reduces the thermal mass of the oven, because less heat gets absorbed by the steel walls.</li>
<li>By reflecting or conducting heat, it helps to eliminate hot and cold spots in the oven.</li>
<li>Good insulation means less energy is needed to reach high temperatures. This means the heating elements can run cooler, further helping to even out temperatures in the oven. For this particular oven, the bottom element runs at a 95% duty cycle without insulation, but just 55% when insulated.</li>
</ol>
Use insulation and use a lot of it! Once you see the great results you get you'll increase the number and complexity of the boards you design. Do it once, and do it right. One is good, more is unnecessary. If the desire is to get an average reading you can connect multiple thermocouples in parallel. Thermocouples generate a micro-voltage that is read by the MAX31856 IC, so wiring them in parallel will give you that average. If the desire is to measure temperature variance around the oven, the best method is to use <a href="http://www.tiptemp.com/Products/Five-Temperature-Circular-Clock-Labels/TLCSEN054-Temperature-Label-5-Level-Clock-Indicator-8.html" target="_blank">Temperature Test Labels</a>, as shown <a href="https://sites.google.com/site/markscontroleo2build/" target="_blank">here</a>. This will give you a more accurate picture of the variance and let you know where the hot and cold areas are in the oven.
<p>Non-convection ovens are inherently left-right temperature balanced. Top-bottom imbalance is expected (and desired). Front-back imbalance will occur because of the wattage and location of the heating elements, type of insulation and the oven door. A convection fan may reduce front-back variation but will also introduce some amount of left-right fluctuation. In almost all ovens, the center will be hotter than the corners.</p>
To convert the toaster oven to a reflow oven, here are the steps we'll be taking:
<ol>
<li><a href="#step1">Plan your build</a></li>
<li><a href="#step2">Open it up</a></li>
<li><a href="#step3">Remove the old controller</a></li>
<li><a href="#step4">Seal the oven</a></li>
<li><a href="#step5">Prepare the relay assembly</a></li>
<li><a href="#step6">Prepare the power supply</a></li>
<li><a href="#step7">Prepare the enclosure</a></li>
<li><a href="#step8">Drill holes</a></li>
<li><a href="#step9">Insulate the inside of the oven</a></li>
<li><a href="#step10">Install the boost element</a></li>
<li><a href="#step11">Install the thermocouple</a></li>
<li><a href="#step12">Seal the holes</a></li>
<li><a href="#step13">Do the high-voltage wiring</a></li>
<li><a href="#step14">Do the low-voltage wiring</a></li>
<li><a href="#step15">Wire up Controleo3</a></li>
<li><a href="#step16">Add the ceramic fiber blanket</a></li>
<li><a href="#step17">Close the oven</a></li>
<li><a href="#step18">Configure Controleo3 and finish the door servo</a></li>
<li><a href="#step19">Add the last bit of insulation</a></li>
<li><a href="#step20">Add the door seal</a></li>
<li><a href="#step21">Install the PCB tray</a></li>
<li><a href="#step22">Run the learning cycle</a></li>
<li><a href="#step23">Run some test cycles</a></li>
<li><a href="#step24">Create your own profiles</a></li>
<li><a href="#step25">Controleo3 tips</a></li>
<li><a href="#step26">Final thoughts</a></li>
</ol>
This build uses high-temperature silicone glue to seal air gaps. This glue is supposed to have a set time of 1 hour, and be fully cured in 24 hours. The reality is that we are applying it rather thick so the curing time is much longer. We recommend at least 24 hours between applying the glue and continuing with the build. In our experience it often takes 3 or 4 days to fully cure.
<p>The average time needed to build this oven is around 8 hours (excluding glue curing time). One customer told us he did it in 5 hours, and one person did it in 7 months. This is the 3rd major revision of this guide. We've been refining our kit contents and build methods to make it easier and quicker to build, and perform better. You're welcome!</p>
<p>The guide is designed to get you quickly into the gluing sections (<a href="#step4">step 4</a> and <a href="#step12">step 12</a>), and allows you to build some sub-assemblies while waiting for the glue to dry. Do not run your oven through its learning cycle or do a reflow until the glue is fully cured.</p>
<p><span>Tip: if you're planning a weekend build you may want to do a little bit on Monday and Wednesday.</span></p>
<p><span>Tip: do not unwrap the white ceramic fiber blanket until <a href="#step16">step 16</a>! Caution is needed.</span></p>
<span>Disclaimer:</span> Converting an oven to a reflow oven will probably invalidate the manufacturer's warranty on the oven. Ovens use high voltage electricity and potentially lethal electric currents. Whizoo (and Peter Easton) cannot be held liable or responsible or will accept any type of liability in any event, in case of injury or even death by building and/or using or misuse of this information posted on this web site. By accessing, reading, and/or printing the articles presented here you agree to be solely responsible as stated in the above disclaimer and exempt Whizoo and/or Peter Easton from any criminal and/or liability suit. Safety is a primary concern when working with high voltage circuits and high-temperature ovens. Play it safe!
<p><span>Tip: click on any photo to see the high-resolution version.</span></p>
<p><span>Tip: take a look at the <a href="https://whizoo.com/hof">Hall of Fame</a> to get inspiration from others.</span></p>
<p><span>Tip: think about ordering an <a href="#step21">aluminum tray</a> if you haven't already done so. One tray is included in the build kit.</span></p>
<p>This should be pretty easy. Make sure the oven is <span>UNPLUGGED</span> and keep track of where all the screws go.</p>
<p><a href="https://whizoo.com/i/c3/b04.jpg"><img src="https://whizoo.com/i/c3/s04.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b05.jpg"><img src="https://whizoo.com/i/c3/s05.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b06.jpg"><img src="https://whizoo.com/i/c3/s06.jpg"></a><br></p>
<p><span>Tip: now is a good time to identify sharp edges in and around the oven. Place tape over them so you don't cut your hands as you work on the oven. Or don't - because the best projects are the ones that involve blood.</span></p>
To keep things safe and to make space for the new controls you should remove all unnecessary parts from the oven. Identify the wiring used by the elements; you'll need to connect them to the SSR's in <a href="#step13">step 13</a>. When cutting wires, leave them as long as possible; they need to be able to reach your SSR's. Save any wire from the oven; it is high-temperature wire that could come in handy later. The top control is removed because that is where the wires to the controller will be fed through.
<p><a href="https://whizoo.com/i/c3/b07.jpg"><img src="https://whizoo.com/i/c3/s07.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b08.jpg"><img src="https://whizoo.com/i/c3/s08.jpg"></a><br></p>
<span>Tip: It doesn't matter how careful you are, this glue is like a sailor on shore leave - it will go everywhere and leave a sticky residue on everything! Protect your workspace with cardboard and paper towels :-)</span>
<p>Ovens come with huge air gaps that will allow hot air to escape. In this step we remove the rear panel of the oven, and seal all the gaps that exist. Then the back panel is replaced.</p>
<p><span>Tip: You can use a light inside the oven to help identify where the gaps are.</span></p>
<p><span>Tip: When smearing glue with your finger, be careful of sharp edges.</span></p>
<p><a href="https://whizoo.com/i/c3/b09.jpg"><img src="https://whizoo.com/i/c3/s09.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b10.jpg"><img src="https://whizoo.com/i/c3/s10.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b11.jpg"><img src="https://whizoo.com/i/c3/s11.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b12.jpg"><img src="https://whizoo.com/i/c3/s12.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b13.jpg"><img src="https://whizoo.com/i/c3/s13.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b14.jpg"><img src="https://whizoo.com/i/c3/s14.jpg"></a></p>
<p>Smear the beads to glue to better seal the gaps, and help the glue cure faster. The front corners of the oven, and the gaps around the heating elements are closed too.</p>
<p><a href="https://whizoo.com/i/c3/b15.jpg"><img src="https://whizoo.com/i/c3/s15.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b16.jpg"><img src="https://whizoo.com/i/c3/s16.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b17.jpg"><img src="https://whizoo.com/i/c3/s17.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b18.jpg"><img src="https://whizoo.com/i/c3/s18.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b19.jpg"><img src="https://whizoo.com/i/c3/s19.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b20.jpg"><img src="https://whizoo.com/i/c3/s20.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b21.jpg"><img src="https://whizoo.com/i/c3/s21.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b22.jpg"><img src="https://whizoo.com/i/c3/s22.jpg"></a></p>
<span>Tip: Do the next few sections while waiting 24 hours or longer for the glue to dry.</span>
<p>Controleo3 supports both mechanical and solid-state relays (SSR). If you don't plan to use your oven much, or if you don't have 2 pennies to rub together you can use <a href="http://www.ebay.com/itm/261966674558" target="_blank">mechanical relays</a>. The contacts in mechanical relays do wear out over time. SSR's have no moving parts so they aren't affected by physical shock, vibration or changes in altitude. They operate silently and produce little electrical interference. They don't produce any arc so they are suitable for hazardous environments. We only use SSR's when building reflow ovens.</p>
<p>When choosing SSR's keep in mind the environment they will operate in (it is hot and the heat sink is relatively small). You should derate the SSR to allow for this. In the case of an SSR controlling power to two 375W elements, the current will be 2 * 375W / 110V = 7 Amps. Choose a 20A SSR to be safe.</p>
<p>The relays are mounted to a thermally conductive piece of metal. Copper has excellent thermal properties, and aluminum is pretty good too. That's why these metals are used as heat sinks and inside computers. Steel is actually <a href="http://en.wikipedia.org/wiki/List_of_thermal_conductivities" target="_blank">not that good</a>. Each SSR can handle 5A without needing a heat sink, and these are getting less than 7A each. When doing a reflow, the SSR's and aluminum plate get slightly warm.</p>
<p>The SSR's in this guide are mounted on a 6" x 6" aluminum sheet that is 1/16" thick. Figure out how this sheet will be mounted in the oven, and how the relays will be mounted on the sheet. Keep the sheet away from the oven wall, and as low as possible (heat rises). Place a thermally conductive pad or paste between the SSR's and the aluminum sheet. Rivets are great for keeping things together but you can use nuts and bolts too. Make sure nothing can come loose because the last thing you need is anything rolling around near high-voltage wiring!</p>
<p><span>Tip: make sure all the oven wires will be able reach the SSR's.</span></p>
<p><span>Tip: (convection ovens) if the aluminum sheet is smaller than used here, or the relays are carrying more current then consider adding active (cooling fan) and passive (heat sink) cooling to ensure the SSR's don't overheat.</span></p>
<p>We make six 3/16" holes for the SSR's, a 3/16" hole for a zip tie (<a href="#step13">step 13</a>) and a 1/4" hole for the wire from the bottom element to pass through. Cut the blue thermal pad into 4 pieces. Remove the plastic cover on both sides, and place it on the back of the SSR's. Trim the pad to fit.</p>
<p><span>Tip: remove burrs on the hole the wire will be fed through. A countersink bit is good for this.</span></p>
<p><a href="https://whizoo.com/i/c3/b23.jpg"><img src="https://whizoo.com/i/c3/s23.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b24.jpg"><img src="https://whizoo.com/i/c3/s24.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b25.jpg"><img src="https://whizoo.com/i/c3/s25.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b26.jpg"><img src="https://whizoo.com/i/c3/s26.jpg"></a></p>
We've decided to mount the 5V power supply for Controleo3 inside the oven. The control chamber (with the relays and power supply) gets warm during a reflow - but nowhere near hot enough to damage the power supply or melt PVC-covered wiring. While you must use high-temperature (silicone or fiberglass) wire inside the oven and for any wires touching the oven walls, ordinary PVC wire can be used in the control chamber.
<p>The power supply needs some wire attached to it. The wire is crimped onto female spade connectors, shoved onto the blades of the power supply and then covered with heat shrink. Don't cut the leads yet - they need to be uneven lengths. Cut the power connector off the end.</p>
<p><span>Tip: before doing this, it might be a good idea to test the power supply and Controleo3. Cut the power connector off, strip the wires and screw them into Controleo3.</span></p>
<p><a href="https://whizoo.com/i/c3/b27.jpg"><img src="https://whizoo.com/i/c3/s27.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b28.jpg"><img src="https://whizoo.com/i/c3/s28.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b29.jpg"><img src="https://whizoo.com/i/c3/s29.jpg"></a></p>
Figure out where Controleo3 will be mounted. Make sure the oven door can still open (don't forget about the servo bracket), and the oven can be put back together!
<p>The servo is attached to the enclosure using a custom Whizoo servo bracket. An aluminum plate used to attached Controleo3 to the oven is also attached. We know, we know, technical drawings showing measurements would be nice, but none of these placements is very critical. The photos should give you a good idea of where everything goes.</p>
<p>The hole in the enclosure is for the wires to pass through, and 7/16" seems to be the right size. A step drill bit makes a nice, clean hole. The holes in the aluminum plate are 9/64" for the bolts, and 1/8" to attach everything to the oven in the next step.</p>
<p>The servo is attached to the bracket using the screws that came with the servo. These screws are pretty weak so consider using other screws you have lying around. The fit is pretty tight so you need the right size Philips screwdriver, and consider using a drill to enlarge the 4 holes in the bracket if necessary. Keep the black M3 screw - you'll need it to attach the servo arm in <a href="#step18">step 18</a>. Screw it into the servo for safekeeping.</p>
<p><span>Tip: real men (and women) would use rivets to attach the aluminum plate to the enclosure.</span></p>
<p><span>Tip: don't underestimate the spatial perception needed to figure out where the aluminum plate goes. It could go on the left or the right, facing up or facing down - but only one combination is correct.</span></p>
<p><span>Tip: take a look at <a href="#step18">step 18</a> to see how the servo will be opening the door.</span></p>
<p><a href="https://whizoo.com/i/c3/b30.jpg"><img src="https://whizoo.com/i/c3/s30.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b31.jpg"><img src="https://whizoo.com/i/c3/s31.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b32.jpg"><img src="https://whizoo.com/i/c3/s32.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b33.jpg"><img src="https://whizoo.com/i/c3/s33.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b34.jpg"><img src="https://whizoo.com/i/c3/s34.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b35.jpg"><img src="https://whizoo.com/i/c3/s35.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b36.jpg"><img src="https://whizoo.com/i/c3/s36.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b37.jpg"><img src="https://whizoo.com/i/c3/s37.jpg"></a></p>
<span>Tip: hopefully the glue is sufficiently dry now that metal shavings don't get stuck in it.</span>
<p>The boost element will be held up by 3/16" x 1-1/2" eye bolts. The ends of the element don't get nearly as hot as the middle, so they eye bolts should be located near the ends of the element.</p>
<p>You'll be making 10 holes:</p>
<ul>
<li>A hole for the wires from the boost element into the control chamber (7/32").</li>
<li>A hole for the wires from the thermocouple probe into the control chamber (3/16").</li>
<li>2 holes for the eye bolts used to support the boost element (3/16").</li>
<li>2 holes for the zip tie that will hold the power supply (7/32").</li>
<li>2 holes to hold the SSR plate in place (9/64"). Remove the screws in that location first.</li>
<li>2 holes in the oven to mount the enclosure (1/16").</li>
</ul>
<p>Firmly attach the SSR plate to the oven. Avoid using the wall to the oven as a mount point. Take care to ensure no metal filings are left behind! When you're done, make sure the wires for the thermocouple and the boost element fit through their respective holes.</p>
<p><span>Tip: use a countersink bit to deburr the holes!</span></p>
<p><span>Tip: resist the temptation to put the <a href="#step16">white ceramic fiber blanket</a> behind the aluminum plate at this time! Yes, it is easier to do it now but you'll be fighting ceramic dust for a long time.</span></p>
<p><a href="https://whizoo.com/i/c3/b38.jpg"><img src="https://whizoo.com/i/c3/s38.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b39.jpg"><img src="https://whizoo.com/i/c3/s39.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b40.jpg"><img src="https://whizoo.com/i/c3/s40.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b41.jpg"><img src="https://whizoo.com/i/c3/s41.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b42.jpg"><img src="https://whizoo.com/i/c3/s42.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b43.jpg"><img src="https://whizoo.com/i/c3/s43.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b44.jpg"><img src="https://whizoo.com/i/c3/s44.jpg"></a></p>
<p><span>Tip: you can remove the enclosure for now and re-install it again in step 14. It makes installing the insulation more difficult.</span></p>
We insulate the inside of the oven to prevent heat from being lost to the steel oven walls. <a href="http://www.designengineering.com/category/catalog/design-engineering-inc/heat-sound-barrier/reflect-gold" target="_blank">Reflect-A-Gold</a> is applied first. This insulation is capable of handling continual temperatures up to 450&deg;C (850&deg;F). The goal of this insulation is to minimize heat loss through the oven door and to reduce the thermal mass of the oven by reducing the amount of heat lost to the steel chassis. Reflect-A-Gold works by reflecting heat back to where it came from.
<p>Make sure to use genuine Reflect-A-Gold, not some fake stuff available from eBay or AliExpress. The insulating properties are very similar but the adhesive on this tape doesn't stand up to the high reflow temperatures as can be seen <a href="https://whizoo.com/i/c/l/rag1.jpg">here</a> and <a href="https://whizoo.com/i/c/l/rag2.jpg">here</a>.</p>
<p>We do a "best effort" when applying the Reflect-A-Gold, and don't cover the whole oven. The loss through the uncovered steel walls is minimal (small surface area), and the ceramic fiber blanket more than makes up for it.</p>
<p><span>Tip: use a bungee (elasticated cord) to hold the oven door open while you place the insulation.</span></p>
<p><a href="https://whizoo.com/i/c3/b45.jpg"><img src="https://whizoo.com/i/c3/s45.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b46.jpg"><img src="https://whizoo.com/i/c3/s46.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b47.jpg"><img src="https://whizoo.com/i/c3/s47.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b48.jpg"><img src="https://whizoo.com/i/c3/s48.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b49.jpg"><img src="https://whizoo.com/i/c3/s49.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b50.jpg"><img src="https://whizoo.com/i/c3/s50.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b51.jpg"><img src="https://whizoo.com/i/c3/s51.jpg"></a><br></p>
<p><a href="http://www.designengineering.com/category/catalog/boom-mat-acoustical-products/floor-tunnel-shield-ii-heat-sound-insulation" target="_blank">Floor &amp; Tunnel Shield II</a> is used to cover most of the interior of the oven. This insulation can handle continual temperatures up to 950&deg;C (1750&deg;F). The aluminum (being an excellent conductor of heat) helps to wick away the heat from the heating elements and spread it evenly around the oven. Care must be taken when putting Floor &amp; Tunnel Shield II above the top element. The adhesive can fail at high temperatures and the Floor &amp; Tunnel Shield II can fall onto the top element and catch fire (we tested this!). There are 4 things we do to alleviate this problem:</p>
<ol>
<li>Use a large piece so if the glue directly above the element fails it will still be held in place.</li>
<li>Make sure the insulation is firmly pressed down, to ensure the adhesion of the glue and maximize the space between the element and the roof of the oven.</li>
<li>A screw is inserted through the Floor &amp; Tunnel Shield II as insurance.</li>
<li>By default, the power to the top element is intentionally limited to 75%. This can be overwritten by the reflow profile file if necessary.</li>
</ol>
<p>Always test-fit the Floor &amp; Tunnel Shield II before peeling the backing off and exposing the adhesive. To place the Floor &amp; Tunnel Shield II, the best method is to make a cut down the center. Peel the plastic backing off and place the 2 pieces further apart so that the middle 1/3 is sticky. Feed the Floor &amp; Tunnel Shield II into position, and firmly press down on the middle. You can then bend up the edges and remove the rest of the backing - and press down the edges.</p>
<p>The largest piece of Floor &amp; Tunnel Shield II goes on the floor of the oven.</p>
<p><span>Tip: (convection ovens) if you need to place 2 pieces of Floor &amp; Tunnel Shield II next to each other, keep in mind how aluminum conducts heat. The pieces should stretch from hot spots (elements) to cool spots (corners). For example, a 2"-wide strip directly beneath a heating element is useless because the heat it absorbs won't be dissipaited.</span></p>
<p><span>Tip: always test-fit the Floor &amp; Tunnel Shield II before peeling the backing off.</span></p>
<p><a href="https://whizoo.com/i/c3/b52.jpg"><img src="https://whizoo.com/i/c3/s52.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b53.jpg"><img src="https://whizoo.com/i/c3/s53.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b54.jpg"><img src="https://whizoo.com/i/c3/s54.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b55.jpg"><img src="https://whizoo.com/i/c3/s55.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b56.jpg"><img src="https://whizoo.com/i/c3/s56.jpg"></a><br></p>
<p>Add spacers (small pieces of Floor &amp; Tunnel Shield II) to the back of the oven. This prevents the tray from being flush with the back of the oven, which helps airflow and will give more consistent results. Don't forget to add a screw (or rivet) as insurance if the adhesive fails. Keep a small piece of Reflect-A-Gold to insulate the around around the thermocouple (<a href="#step19">step 19</a>).</p>
<p><a href="https://whizoo.com/i/c3/b57.jpg"><img src="https://whizoo.com/i/c3/s57.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b58.jpg"><img src="https://whizoo.com/i/c3/s58.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b59.jpg"><img src="https://whizoo.com/i/c3/s59.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b60.jpg"><img src="https://whizoo.com/i/c3/s60.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b61.jpg"><img src="https://whizoo.com/i/c3/s61.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b62.jpg"><img src="https://whizoo.com/i/c3/s62.jpg"></a><br></p>
The boost element is a 110V 350W resistive heating element normally used for heating molds. It helps installation to have one eye bolt fit snuggly on the element, but isn't absolutely necessary because the element will be held in place by the wires glued to the side of the oven. You can enlarge or reduce the size of the eye hole using a vise or hammer. Make a hole through the Floor &amp; Tunnel Shield II using the holes you made in the oven earlier as a guide. An awl is the right tool for this, but any sharp, pointy object will work. Put the element through the eye bolts and then put the bolts through the holes. Make sure everything is bolted down securely.
<p><a href="https://whizoo.com/i/c3/b63.jpg"><img src="https://whizoo.com/i/c3/s63.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b64.jpg"><img src="https://whizoo.com/i/c3/s64.jpg"></a><br></p>
The "k-type" thermocouple probe is fiberglass insulated and can measure up to 550&deg;C. This particular probe is used because it has low-mass tip so it responds very quickly to changes in temperature. The thermocouple should measure the temperature experienced by the solder so the best way to mount the thermocouple is to use aluminum tape and tape it to the board being reflowed. This is a really good method in theory, but in practice it falls short. It is difficult to place the probe on the PCB (even a sacrificial PCB) and get consistent results. The thermocouple could be over a via, or there may be a tiny air gap between it and the PCB. There's also the ever-present danger of bumping components when trying to place the thermocouple.
<p>A better method is to recognize that the thermal mass of the oven is largely unchanged between runs. It is best to place the thermocouple in a location that should reflect the board temperature and is away from any direct heat from the heating elements. We have built ovens and reflowed boards using both placement methods and we recommend this method without reservation.</p>
<p>You should have 3" or 4" of the thermocouple inside the oven.</p>
<p><span>Tip: masking tape is used to hold the thermocouple in place while the glue dries. This <span>must</span> be removed before closing the oven otherwise it will burn!</span></p>
<p><a href="https://whizoo.com/i/c3/b65.jpg"><img src="https://whizoo.com/i/c3/s65.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b66.jpg"><img src="https://whizoo.com/i/c3/s66.jpg"></a><br></p>
This is the last bit of gluing. Seal the holes around the thermocouple and the boost element wires. Don't forget to remove the piece of tape once the glue is dry.
<p><a href="https://whizoo.com/i/c3/b67.jpg"><img src="https://whizoo.com/i/c3/s67.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b68.jpg"><img src="https://whizoo.com/i/c3/s68.jpg"></a><br></p>
It is important to get your high-voltage wiring right! If your oven is 220V your elements may be wired in series, not parallel. If you are not 100% confident dealing with electricity, please consult an electrician. You can also ask questions using the feedback form below.
<p>The "hot" wire goes to each of the SSR's and to the 5V power supply. The "neutral" wire goes to one end of each heating element and also goes to the 5V power supply. The other end of each element is also wired to a SSR. Use zip ties to keep the wiring neat.</p>
<p><span>Tip: label the SSR's so that you know which SSR controls which heating element.</span></p>
<p><span>Tip: if you need more wire, you may be able to take some from the boost element or the blue silicone wire.</span></p>
<p><a href="https://whizoo.com/i/c/l/schematic.png"><img src="https://whizoo.com/i/c/s/schematic.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b69.jpg"><img src="https://whizoo.com/i/c3/s69.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b70.jpg"><img src="https://whizoo.com/i/c3/s70.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b71.jpg"><img src="https://whizoo.com/i/c3/s71.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b72.jpg"><img src="https://whizoo.com/i/c3/s72.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b73.jpg"><img src="https://whizoo.com/i/c3/s73.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b74.jpg"><img src="https://whizoo.com/i/c3/s74.jpg"></a><br></p>
The wires coming from the oven are:
<ul>
<li>Wire from the 5V power supply.</li>
<li>Wire for 3 SSR's. The relays have common positive, so we use a 4-conductor wire for 3 SSR's. The wire needed won't be carrying much current (max 10ma @ 5V per SSR) so even something as thin as a USB cable can be used. Mechanical relays need around 50ma @ 5V each, so again the wire doesn't need to be a heavy gauge.</li>
<li>The thermocouple wires.</li>
</ul>
The thermocouple generates a micro-voltage based on the temperature of the oven. Running the thermocouple close to high-voltage AC wires will induce noise into the temperature reading. Keep as large a gap as possible between the high and low voltage wires. Make sure the clearance exists behind the aluminum SSR plate too; sometimes the thermocouple rests against the top or bottom element wires.
<p>For the low-voltage wiring, we'll be using each conductor as follows:</p>
<ul>
<li>Controleo3 output 1 = black wire = SSR for bottom element</li>
<li>Controleo3 output 2 = blue wire = SSR for top element</li>
<li>Controleo3 output 3 = yellow wire = SSR for boost element</li>
<li>Controleo3 output "+" = red wire = common positive for all SSR's</li>
</ul>
Feed the wires through the grommet and through the braided sleeving. To keep things neat and secure, we use 10" of 1/4" braided sleeving along with zip ties.
<p><span>Tip: the 3rd photo below shows the best method to feed the wires through the grommet and enclosure.</span></p>
<p><span>Tip: make sure the thermocouple wire isn't close to high-voltage wire!</span></p>
<p><a href="https://whizoo.com/i/c3/b75.jpg"><img src="https://whizoo.com/i/c3/s75.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b76.jpg"><img src="https://whizoo.com/i/c3/s76.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b77.jpg"><img src="https://whizoo.com/i/c3/s77.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b78.jpg"><img src="https://whizoo.com/i/c3/s78.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b79.jpg"><img src="https://whizoo.com/i/c3/s79.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b80.jpg"><img src="https://whizoo.com/i/c3/s80.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b81.jpg"><img src="https://whizoo.com/i/c3/s81.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b82.jpg"><img src="https://whizoo.com/i/c3/s82.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b83.jpg"><img src="https://whizoo.com/i/c3/s83.jpg"></a><br></p>
Next we'll get power and temperature to Controleo3. The good news is that Controleo3 has reverse polarity protection - but still, don't reverse the power wires. More good news is that you won't cause damage if the thermocouple wires are reversed either. If they are reversed, the temperature will go down when it should go up - or vice versa. So go ahead all you color-blind people - you have a 50% chance of getting it right!
<p>Avoid repeatedly bending wires past their breaking point. Hopefully you left your wires long so you can strip them again if necessary.</p>
<p>Remove the plastic film protecting the LCD display.</p>
<p><span>Tip: filing the panelization tab on the Controleo 3 PCB (near the USB port) might allow the USB port to protrude further.</span></p>
<p><span>Tip: unscrew all the screw terminals you plan to use now.</span></p>
<p><span>Tip: take care you don't have any "careless whiskers" when putting wires into the screw terminals.</span></p>
<p><a href="https://whizoo.com/i/c3/b84.jpg"><img src="https://whizoo.com/i/c3/s84.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b85.jpg"><img src="https://whizoo.com/i/c3/s85.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/cl01.png"><img src="https://whizoo.com/i/c3/cs01.png"></a></p>
<p><span>Test 1 - Check 5V power and thermocouple</span><br>
Make sure your high-voltage wiring is safe, then plug in the oven. Controleo3 should receive power and start up in 3 seconds. Look at the temperature reading on the display. It should read room temperature (around 25&deg;C). Carefully place your hand inside the oven and touch the tip of the thermocouple. The temperature should rise - unless you have cold hands. If the direction of temperature change isn't correct then reverse the red and green wires.</p>
<p><span>&gt;&gt;&gt; Unplug the oven when you're done. &lt;&lt;&lt;&lt;</span></p>
<p><span>Tip: you might want to take a photo of your wiring, in case you need to refer back to it.</span></p>
<p>Finish screwing the remaining 4 wires into Controleo3, and close the enclosure. Keep an eye on the USB port - that has to be inserted first. When putting the lid on the enclosure, take care not to overtighten the screws. The lid can press down on the touch screen and interfere with touch readings.</p>
<p><a href="https://whizoo.com/i/c3/b86.jpg"><img src="https://whizoo.com/i/c3/s86.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b87.jpg"><img src="https://whizoo.com/i/c3/s87.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b88.jpg"><img src="https://whizoo.com/i/c3/s88.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b89.jpg"><img src="https://whizoo.com/i/c3/s89.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/cl02.png"><img src="https://whizoo.com/i/c3/cs02.png"></a>&nbsp; <a href="https://whizoo.com/i/c3/cl03.png"><img src="https://whizoo.com/i/c3/cs03.png"></a></p>
<p><span>Test 2 - Check SSR operation</span><br>
Make sure your high-voltage wiring is safe, then plug in the oven. Controleo3 should receive power and show the Settings menu. Tap "Test" and then the "Turn On" button. This button should toggle the bottom element on and off. Take a look at the LED indicator on the SSR, and keep an eye on the element. Expect to hear some buzzing from the elements as they start to heat up. Toggle power on and off until you are satisfied that the bottom element is being controlled as expected.</p>
<p>Tap the blue arrow on the lower-right of the screen to advance to output 2 - which should be the top element. Make sure that works, then advance to output 3 (the boost element). You should see the temperature rise, as the elements generate heat</p>
<p><span>&gt;&gt;&gt; Unplug the oven when you're done. &lt;&lt;&lt;&lt;</span></p>
<p><img src="https://whizoo.com/i/c3/test.gif"></p>
Ceramic fibers are very fine and sharp. They don't penetrate though thick skin, but can irritate thin tissue like eyes, nostrils and lungs. If you exercise precautions you won't have any problems. The blanket has amazing insulating properties, and the top of your oven will be barely warm during a reflow.
<p>Things to keep in mind:</p>
<ul>
<li>Unwrap the ceramic fiber blanket just before you use it - not earlier.</li>
<li>Clean up your workspace immediately after applying the blanket and closing the oven</li>
<li>If possible, wear a face mask (we recommend using N95 masks)</li>
<li>Wear shoes. Our preference is flip-flops that we can easily wash afterwards</li>
<li>Take care not to touch your eyes, lips or ears</li>
<li>Keep the area free of drinks, and use the restroom (toilet) before starting</li>
<li>Work in a draft-free environment. Don't use fans</li>
<li>Make slow movements to minimize dust</li>
<li>Make sure you have all your tools and parts close by (you need to close up the oven too)</li>
<li>Make sure pets, kids and neglected spouses don't come into your work area</li>
<li>Immediately after working with the blanket you should take a shower and wash your clothing</li>
</ul>
Scared? Don't be. Ceramic fiber is easy to work with as long as you take a few precautions.
<p><span>Tip: ceramic fiber is tough on scissors. Use an old pair you don't care about.</span></p>
<p><a href="https://whizoo.com/i/c3/b90.jpg"><img src="https://whizoo.com/i/c3/s90.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b91.jpg"><img src="https://whizoo.com/i/c3/s91.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b92.jpg"><img src="https://whizoo.com/i/c3/s92.jpg"></a></p>
<p><span>Tip: An alternative to cutting the ceramic blanket to allow the cover to fit is to modify the cover. One customer used a Dremel, but you could probably use tin snips too.</span></p>
<p><a href="https://whizoo.com/i/c3/b93.jpg"><img src="https://whizoo.com/i/c3/s93.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b94.jpg"><img src="https://whizoo.com/i/c3/s94.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b95.jpg"><img src="https://whizoo.com/i/c3/s95.jpg"></a><br></p>
Well, good luck! Manufacturing tolerances for these ovens seems to be very tight and it isn't easy getting them back together again. At least you know that it is possible, because you took it apart.
<p><span>Tip: before closing the oven make a last check for loose nuts, bolts, screws, wires and metal shavings. Use zip ties to keep wires in place. Make sure the chassis screws you're about to insert don't pierce any work you've just done.</span></p>
<p><span>Tip: make sure all tape has been removed. It will burn at high temperatures.</span></p>
<p><span>Tip: you might want to take a few photos of your masterpiece!</span></p>
<p><a href="https://whizoo.com/i/c3/b96.jpg"><img src="https://whizoo.com/i/c3/s96.jpg"></a><br></p>
Turn on the oven, and navigate to Setup (Settings -&gt; Setup). Configure the outputs as you've wired them in step 14 and 15. Continue to navigate through the setup screens to the door open position.
<p><img src="https://whizoo.com/i/c3/setup.gif"></p>
<p>It should show 90&deg; - this is the mid-point in the servo movement. If it doesn't show 90&deg; then press the "+" or "-" button until it does. Attach the servo arm to the servo disc using four M3 screws. Mount the arm to the servo in what you think will be the mid-point of it's movement.</p>
<p><a href="https://whizoo.com/i/c3/b97.jpg"><img src="https://whizoo.com/i/c3/s97.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b98.jpg"><img src="https://whizoo.com/i/c3/s98.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b99.jpg"><img src="https://whizoo.com/i/c3/s98.jpg"></a><br></p>
<p>Take a 3" x 1/2" piece of aluminum and drill 2 holes in it using a 1/8" drill bit. Give it a 90&deg; bend as shown in the photos. Temporarily tape it to the oven handle while playing with the open/close motion (by going between the "Door open position" and "Door closed position" screens) of the door, and adjusting the aluminum to work. Drill 2 pilot holes in the oven handle using a small 1/16" or 5/64" drill bit. Attach the aluminum to the handle. The oven door should open around 1". That is enough to provide sufficient cooling after the reflow.</p>
<p><a href="https://whizoo.com/i/c3/b100.jpg"><img src="https://whizoo.com/i/c3/s100.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b101.jpg"><img src="https://whizoo.com/i/c3/s101.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b102.jpg"><img src="https://whizoo.com/i/c3/s102.jpg"></a><br></p>
<p>The door servo is always moved to the closed position when the oven powers up. Typically, the door will be opened slowly soon after the maximum temperature is reached, and closed again when the oven cools to 50&deg;C. You can change that behavior in the reflow profile though.</p>
Hopefully the glue on the thermocouple is fully cured now? Place a piece of Reflect-A-Gold on the wall of the oven. Position the thermocouple as shown in the photos. The exact placement isn't important; it is more important that it doesn't move between reflows so that you get consistent results.
<p><span>Tip: if you have any left-over Reflect-A-Gold you can place it on the steel oven walls now.</span></p>
<p><a href="https://whizoo.com/i/c3/b107.jpg"><img src="https://whizoo.com/i/c3/s107.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b108.jpg"><img src="https://whizoo.com/i/c3/s108.jpg"></a><br></p>
To stop air loss through the door we place Nomex high-temperature gasket around the door. The Nomex should be cut in half.
<p><a href="https://whizoo.com/i/c3/b103.jpg"><img src="https://whizoo.com/i/c3/s103.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b104.jpg"><img src="https://whizoo.com/i/c3/s104.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b105.jpg"><img src="https://whizoo.com/i/c3/s105.jpg"></a><br>
<a href="https://whizoo.com/i/c3/b106.jpg"><img src="https://whizoo.com/i/c3/s106.jpg"></a><br></p>
The PCB tray is the <span>single most important part</span> of a reflow oven. Good heat distribution is paramount to a successful reflow and the tray the most important part of the equation. The trays that typically come with ovens are made from steel (if it is magnetic it is steel), which you already know is <a href="http://en.wikipedia.org/wiki/List_of_thermal_conductivities" target="_blank">not good</a>. Using the rack is like going to the beach wearing a bikini and expecting a "no-lines" tan!
<p>For this oven we're using a custom cut 1/32" thick 8"x10-5/8" sheet of 5052 aluminum (get them <a href="http://www.ebay.com/itm/322068823844" target="_blank">here</a>. Todd can also do custom trays) that exactly fits our oven. 1/32" is the right thickness; any thicker and we're just adding thermal mass and any thinner it will lose rigidity. Any metal with a good thermal conductivity will work.</p>
<p>Another good approach - especially for convection ovens - is to use the rack that came with the oven but remove most of the wires to reduce thermal mass. A thin aluminum or copper sheet can be placed directly on top of this modified rack. Convection ovens will need good airflow around the metal sheet, so make holes around the edges if necessary.</p>
<p><span>Tip: make sure there is airflow around the tray otherwise Controleo3 will be unaware of how much heat is coming from the bottom of the oven.</span></p>
<p><a href="https://whizoo.com/i/c3/b109.jpg"><img src="https://whizoo.com/i/c3/s109.jpg"></a>&nbsp; <a href="https://whizoo.com/i/c3/b110.jpg"><img src="https://whizoo.com/i/c3/s110.jpg"></a><br></p>
Controleo3 uses a PID algorithm to follow the reflow curve. The most difficult part of PID is tuning the constants to work well with your oven. This is where Controleo3's learning cycle comes in; it will run some tests on your oven to determine the power and time needed to adjust the temperature. It takes around 1 hour to run this cycle.
<p>Another metric to come out of this learning cycle is that it will let you know how well your reflow oven has been built, and if it is lacking in any area. You can read more about this <a href="https://whizoo.com/intelligent">here</a>. If your oven's performance is poor, you might be able to <a href="https://whizoo.com/intelligent#troubleshoot">improve it</a>.</p>
<p>The learning cycle <span>must</span> be run with the oven with its typical reflow load. In other words, the aluminum tray must be inside the oven. You can add a PCB or two if you like.</p>
<p>Anytime the oven is used, it must be done in a <span>well-ventilated area</span>. Initially you'll be getting some off-gassing from the adhesives used in the insulation but you'll also get fumes from the components, the PCB and the flux in the solder paste.</p>
<p>If you want to monitor your oven's performance, you can install Arduino's <a href="https://www.google.com/search?q=Arduino+Serial+Monitor" target="_blank">Serial Monitor</a> software on your PC and monitor the status messages using the Arduino Serial Monitor.</p>
<p><span>Tip: always use your oven in a well-ventilated area.</span></p>
Use scrap PCB's and components you no longer need. Place some solder paste on the boards and some components. A simple test is the "yellow capacitor" test as documented in a Kickstarter <a href="https://www.kickstarter.com/projects/1471240030/controleo2-reflow-oven/posts/1053232" target="_blank">update</a>. A much better method is to use <a href="http://www.tiptemp.com/Products/Five-Temperature-Circular-Clock-Labels/TLCSEN054-Temperature-Label-5-Level-Clock-Indicator-8.html" target="_blank">Temperature Test Labels</a>, as shown <a href="https://sites.google.com/site/markscontroleo2build/" target="_blank">here</a>.
<p>Place boards in a 3x3 grid on the oven tray. This will help you determine if you have any hot or cold spots in your oven. The corners typically run a little cooler and the center a bit hotter. Do a reflow and examine each board in turn. Was there sufficient heat for the solder reflow to happen, but not too much that the boards warped or components were burned?</p>
<p>If not using the door servo, when instructed to open the oven door open it an inch or two. Be careful not to let the cool down rate exceed 6&deg;C/second; the optimal rate is 2&deg;C/second. Once the oven cools to 100&deg;C the boards can be removed, but you can leave them in longer so you don't need gloves to handle them.</p>
You can create your own reflow profiles by following this <a href="https://whizoo.com/profiles">guide</a>. Here are some tips to help you get the most out of Controleo3:
<p><span>1. Take a screenshot</span><br>
To take a screenshot, tap 3 times in the top left corner of the screen. If there is a SD card, a bitmap of the screen will be written to the card. Writing the bitmap takes around 10 seconds, and Controleo3 will freeze during this time (so avoid doing this during a reflow!). You'll hear a beep every second to let you know something is happening.</p>
<p><span>2. Touchscreen</span><br>
The touchscreen is calibrated as part of the final test procedures. If you feel it isn't accurate enough, you can recalibrate it. Go to Settings -&gt; Stats -&gt; Reset -&gt; Touch Calibration. You should use a stylus for better accuracy.<br>
If the touchscreen isn't working at all, try loosening the enclosure lid screws or removing the lid completely. The pressure from the lid can push down on the touchscreen making it unresponsive.</p>
<p><span>3. Tap-and-hold</span><br>
Controleo3 supports tap-and-hold. For example, when setting the bake temperature you can tap-and-hold the "+" arrow to quickly increase the temperature.</p>
<p><span>4. Celsius / Farenheit toggle</span><br>
Reflow temperatures are always given in Celsius, but if you're ever curious what the oven temperature is in Farenheit you can tap on the temperature in the top-right corner of the screen.</p>
<p><span>5. Removing the SD card</span><br>
Reading from or writing to the SD card is unbuffered, so the SD card can be removed without pressing an "eject" button. It feels weird, but you're going to like this a lot!</p>
<p><span>6. Inside information</span><br>
There is a lot more going on behind the scene that what is displayed on the screen. Controleo3 continuously dumps information out of the USB port. To see it, attach a USB cable to your PC and run Arduino's <a href="https://www.google.com/search?q=Arduino+Serial+Monitor" target="_blank">Serial Monitor</a>. You need the Arduino software on your PC anyway to be able to do software updates.</p>
<p><span>7. Troubleshooting reflow problems</span><br>
If you're having problems with your oven's performance you can find helpful information <a href="https://whizoo.com/intelligent#troubleshoot">here</a>.</p>
<p><span>8. Stay informed</span><br>
Join our <a href="https://whizoo.com/about">mailing list</a> to be notified when new software for Controleo3 is available</p>
<span>Reflow process</span><br>
There are a number of guides on how to do this, and a quick search of YouTube or the Internet will produce results. Click <a href="https://www.kamijin.com/soldering-components-with-reflow" target="_blank">here</a> for a pretty good one.
<p><span>Boards and paste</span><br>
<a href="https://www.oshpark.com" target="_blank">OSHPark</a> produces excellent quality boards and delivers them quickly. For most prototype boards, having a stencil reduces assembly time and improves soldering so much less rework is needed. We love stainless steel stencils, and have had success using <a href="https://www.oshstencils.com" target="_blank">OSH Stencils</a>. For larger quantities we often order boards and stencils from <a href="https://www.pcbway.com" target="_blank">PCBWay</a> (use coupon code CD16CBDE) or <a href="http://smart-prototyping.com" target="_blank">Smart Prototyping</a> in China.</p>
<p>We personally use Chip Quik SAC305 lead-free paste and we're happy with the results. From feedback we've received, any no-clean lead-free paste appears to be as good as any other. Paste freshness is important; the flux in the paste will degrade over time. Keep it refrigerated per the manufacturer's instructions. Fine-pitch components and especially BGA's will need the freshest and best quality paste. If you're doing a low-frequency design with large component pads you can probably use cheaper, older paste.</p>
<p><span>Insight into the solder paste reflow curve</span><br>
Solder paste manufacturers try to sell paste to contract manufacturers with conveyor belt reflow ovens. Time is money for them, so the reflow graph typically shows the shortest reflow duration under ideal conditions. All datasheets state something along the lines of "use this graph as a starting point for process optimization". Another point to keep in mind is that your oven doesn't get instantly hot when you turn it on, so it is best to zero-base profile graphs to 50&deg;C or 75&deg;C for a better comparison. Taking a longer time at lower (pre-soak) temperatures is almost inconsequential to the process.</p>
<p><span>Improving your oven's performance</span><br>
Your oven might do perfect reflows immediately, but it'll probably take a few tuning runs before that happens. You should take a closer look at your <a href="https://whizoo.com/intelligent">oven's performance numbers</a>. You might need to modify the <a href="https://whizoo.com/profiles">profile</a> to suit your paste and your oven.</p>
<p><span>Open source software</span><br>
The source code for Controleo3 is <a href="https://github.com/engineertype/Controleo3" target="_blank">open-source</a>, so if you'd like to improve this we'd love to share your efforts with our users!</p>
If you have photos or video you'd like to share on our <a href="https://whizoo.com/hof">Hall of Fame</a> then please let us know. If there is anything we can do to improve this guide let us know that too.
<p><span>Paid Advertisement</span><br>
<span>Don't forget to clean up after you're finished with your <a href="https://www.osha.gov/SLTC/cleaningindustry/standards.html" target="_blank">project</a>. If you keep your workshop or office clean as you go it should be quick <a href="https://comptroller.texas.gov/taxes/publications/94-111.php" target="_blank">job</a> to clean up. Of course hiring a <a href="http://www.commercialofficecleaning.com/" target="_blank">commercial cleaning company</a> is always an option to help maintain your home or office.</span></p>
<p>cheap reflow oven, prototype reflow oven, reflow soldering oven, solder reflow oven, reflow solder oven, pcb reflow oven, solder oven, infrared reflow oven, low cost reflow oven, soldering oven, mini reflow oven, table top reflow oven, bga reflow oven, small reflow oven, oven soldering, reflow oven price, arduino reflow controller, arduino reflow oven, arduino reflow oven controller, arduino smd reflow, build a reflow oven, reflow oven arduino, reflow oven, toaster reflow oven, convection reflow oven, diy reflow oven, reflow toaster oven, zallus reflow, zallus reflow controller, build a reflow oven, homemade reflow oven, zallus, diy toaster reflow oven, zallus controller, t962, t962 reflow oven, t962 oven, t962a, t962a reflow oven, t962a oven, smd reflow, smt reflow oven, smt reflow, smd reflow oven, lead free reflow oven, lead free reflow, high temperature reflow, reflow oven controller, reflowster, benchtop reflow oven, toaster oven reflow, diy toaster oven.</p>
<a href="https://whizoo.com/thegift"><img border="0" width="10" src="https://whizoo.com/i/n/transparent.gif" height="10"></a>
<p>&copy; Whizoo 2019</p>
</div>
</div>
<br>
<br>
<br>
<br>
<br>
<br>
via whizoo.com https://whizoo.com<br>
<br>
July 24, 2019 at 11:06AM
